mmvb_backend/common/utils.py

```diff
@@ -90,6 +99,10 @@ def import_modules(package_name):
 
 
 def perform_request(url, retries=DEFAULT_MAX_RETRIES, timeout=DEFAULT_TIMEOUT):
+    """
+    Performs an http GET request to a given url respecting the maximum amount
```

Comment:
```suggestion
    Performs a http GET request to a given url respecting the maximum amount
```

Reply:
Maximum amount of what?

Reply:
"of retries..." the docstring should continue in the next line